scripts/library.sh

```diff
@@ -340,7 +340,7 @@ function report_go_test() {
   # Run tests in verbose mode to capture details.
   # go doesn't like repeating -v, so remove if passed.
   local args=" $@ "
-  local go_test="go test -race -v ${args/ -v / }"
+  local go_test="go test -v ${args/ -v / }"
```

Comment:
This is a breaking change. Unit tests won't run with `-race` anymore and it will allow race conditions to slip into code. If -race is a problem for the load tests, I suggest making this a helper function or handling a custom `-norace` parameter.

Reply:
Actually, I checked and no one is using `report_go_test` directly. It's safe to remove `-race` from here and add it to `default_unit_test_runner()` in `presubmit-tests.sh` and `go_test_e2e()` here.

Reply:
Yeah I was also checking it :-)
Done.